maquetado/scss/styles.scss

```
@@ -3,13 +3,17 @@
 }
 
 .book-container {
-  padding: 10px;
+  background-color: $white;
+  box-shadow: 4px 4px 10px 0 $black-05;
+  max-width: 1000px;
+  padding: 20px;
+  width: 100%;
 }
 
 .book-cover {
   height: 368px;
-  width: 261px;
   margin-right: 40px;
+  width: 261px;
```

Remember the style guide here, this is greater than 250px: 

```suggestion
width: 100%;
max-width: 261px;
```